docs/source/tools/trigger-service/index.rst

```diff
@@ -14,120 +14,139 @@ Trigger Service
 
 The :ref:`running-a-no-op-trigger` section shows a simple method using the ``daml trigger`` command to arrange for the execution of a single trigger. Using this method, a dedicated process is launched to host the trigger.
 
-Complex workflows can require running many triggers for many parties and at a certain point, use of ``daml trigger`` with its process per trigger model becomes unwieldy. The Trigger Service provides the means to host multiple triggers for multiple parties running against a common ledger in a single process and provides a convenient interface for starting, stopping and monitoring them.
+Complex workflows can require running many triggers for many parties and at a certain point, use of ``daml trigger`` with its process-per-trigger model becomes unwieldy. The trigger service provides the means to host multiple triggers for multiple parties running against a common ledger in a single process and provides a convenient interface for starting, stopping and monitoring them.
 
-The Trigger Service is a ledger client that acts as an end-user agent. The Trigger Service intermediates between the ledger and end-users by running triggers on their behalf. The Trigger Service is an HTTP service. All requests and responses use JSON to encode data.
+The trigger service is a ledger client that acts as an end-user agent. The trigger service intermediates between the ledger and end-users by running triggers on their behalf. The trigger service is an HTTP service. All requests and responses use JSON to encode data.
 
 Starting the Trigger Service
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+****************************
 
-In this example, it is assumed there is a sandbox ledger running on port 6865 on localhost.
+In this example, it is assumed there is a Ledger API server running on port 6865 on `localhost`.
 
 .. code-block:: bash
 
    daml trigger-service --config trigger-service.conf
 
-where the corresponding config would be
-
-The required config would look like
+The following snippet provides an example of what a possible `trigger-service.conf` configuration file could look like,
+alongside a few annotations with regards to the meaning of the configuration keys and possibly their default values.
 
 .. code-block:: none
 
     {
-      //dar file containing the trigger
+      // Mandatory. Paths to the DAR files containing the code executed by the trigger.
```

Comment:
I followed the following approach for edits here:
- document if a field is mandatory
- if it is not
  - if it doesn't have a default, mention it's optional
  - if it has a default, just mention it -- if something has a default, it must be optional
- one space after the comment token for descriptions
- no space after the comment token for values that have been commented out
- consistent capitalization